src/blocking/spi.rs

```diff
@@ -29,26 +57,59 @@ pub trait WriteIter<W> {
         WI: IntoIterator<Item = W>;
 }
 
-/// Blocking transfer
-pub mod transfer {
+/// Blocking simultaneous read+write with separate buffers
+pub mod read_write {
+    /// Default implementation of `blocking::spi::ReadWrite<W>` for implementers of
+    /// `nonblocking::spi::FullDuplex<W>`
+    pub trait Default<W>: crate::nb::spi::FullDuplex<W> {}
+
+    impl<W, S> crate::blocking::spi::ReadWrite<W> for S
+    where
+        S: Default<W>,
+        W: Clone + core::default::Default,
+    {
+        type Error = S::Error;
+
+        fn read_write(&mut self, read: &mut [W], write: &[W]) -> Result<(), S::Error> {
+            for i in 0..core::cmp::max(read.len(), write.len()) {
+                let word_out = if i < write.len() {
+                    write[i].clone()
+                } else {
+                    W::default()
```

Comment:
The value sent here should be documented.

Reply:
Done! I've added to the `Default` docs